src/web/app/views/dialogs/Prompt.tsx

```diff
@@ -56,7 +56,7 @@ function Prompt({
 
   return (
     <Modal>
-      <div className={classNames('prompt-dialog-container', 'animate__animated', 'animate__bounceIn')}>
+      <div id="addparameter" className={classNames('prompt-dialog-container', 'animate__animated', 'animate__bounceIn')}>
```

Comment:
this is a general component, so id like `prompt-container` is more proper

Reply:
finished